notebooks/ch12/LR_Case_Study.md

```diff
@@ -0,0 +1,792 @@
+        text-align: right;
+    }
+
+    .dataframe thead th {
+        text-align: left;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>Bias</th>
+      <th>Urban population</th>
+      <th>Late births</th>
+      <th>Wine</th>
+      <th>Liquor</th>
+      <th>Cirrhosis</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>1</td>
+      <td>44.0</td>
+      <td>33.2</td>
+      <td>5</td>
+      <td>30</td>
+      <td>41.2</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>1</td>
+      <td>43.0</td>
+      <td>33.8</td>
+      <td>4</td>
+      <td>41</td>
+      <td>31.7</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>1</td>
+      <td>48.0</td>
+      <td>40.6</td>
+      <td>3</td>
+      <td>38</td>
+      <td>39.4</td>
+    </tr>
+    <tr>
+      <th>...</th>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+    </tr>
+    <tr>
+      <th>43</th>
+      <td>1</td>
+      <td>44.0</td>
+      <td>35.3</td>
+      <td>3</td>
+      <td>32</td>
+      <td>51.6</td>
+    </tr>
+    <tr>
+      <th>44</th>
+      <td>1</td>
+      <td>63.0</td>
+      <td>39.3</td>
+      <td>8</td>
+      <td>40</td>
+      <td>55.7</td>
+    </tr>
+    <tr>
+      <th>45</th>
+      <td>1</td>
+      <td>58.0</td>
+      <td>43.8</td>
+      <td>13</td>
+      <td>57</td>
+      <td>55.5</td>
+    </tr>
+  </tbody>
+</table>
+<p>45 rows × 6 columns</p>
+</div>
+
+
+
+Looks like it worked! We now have 45 rows instead of 46 rows. 
+
+## Data Visualization
+
+If we are going to try to create a _linear_ model, we should make sure that all of our predictor variables have a somewhat linear relationship with the target variable of Cirrhosis. Let's plot every predictor variable individually against Cirrhosis.  
+
+
+```python
+# HIDDEN
+plt.scatter(drinking_data['Urban population'], drinking_data['Cirrhosis'])
+plt.title("Urban population vs. Cirrhosis")
+plt.xlabel("Urban population")
+plt.ylabel("Cirrhosis");
+```
+
+
+![png](LR_Case_Study_files/LR_Case_Study_25_0.png)
+
+
+This looks like a reasonably linear relationship.
+
+
+```python
+# HIDDEN
+plt.scatter(drinking_data['Late births'], drinking_data['Cirrhosis'])
+plt.title("Late births vs. Cirrhosis")
+plt.xlabel("Late births")
+plt.ylabel("Cirrhosis");
+```
+
+
+![png](LR_Case_Study_files/LR_Case_Study_27_0.png)
+
+
+It looks like we have an outlier for late births values. Late births and cirrhosis seem to have a pretty strong linear relationship, but because this point is not close to the linear trend at all, this outlier would negatively affect our linear regression model. Thus, we should take this row out of our dataset and investigate later whether this was an error in our dataset or an actual recorded value (not in this notebook). 
```

Comment:
Not sure about keeping "and investigate later whether this was an error in our dataset or an actual recorded value (not in this notebook)."